language/stdlib/modules/LibraAccount.move

```diff
@@ -188,6 +201,25 @@ module LibraAccount {
         );
     }
 
+    public fun initialize_writeset_manager(
```

Comment:
The previous version of this code had a spec. Can that be pulled over from the old module as well?